docs/README.md

```diff
@@ -2,137 +2,139 @@
 
 Crossplane is an open source multicloud control plane to manage your
 cloud-native applications and infrastructure across environments, clusters,
-regions and clouds. It enables provisioning and full-lifecycle management
- of applications and managed services from your choice of cloud using `kubectl`.
+regions and clouds. It enables provisioning and full-lifecycle management of
+applications and managed services from your choice of cloud using `kubectl`.
 
 Crossplane can be installed into an existing Kubernetes cluster to add managed
 service provisioning or deployed as a dedicated control plane for multi-cluster
 management and workload scheduling.
 
 Crossplane enables the community to build and publish Stacks to add more clouds
 and cloud services to Crossplane with support for out-of-tree extensibility and
-independent release schedules. Crossplane includes Stacks for [GCP][stack-gcp], 
+independent release schedules. Crossplane includes Stacks for [GCP][stack-gcp],
 [AWS][stack-aws], and [Azure][stack-azure] today.
```

Comment:
Should we add `stack-rook` here? Might also be nice to add something like: "There are also community driven stacks that are owned by other organizations, such as [`stack-packet`](https://github.com/packethost/stack-packet)."

Reply:
I think so.

In this area I was also thinking it feels a little weird to call out that we support "out-of-tree extensibility". Seems like something that anyone who was around when we _didn't_ support that might be interested in, but that is kind of a given in context now.